### ISSUE TITLE
1.9.1: gtkfilechooser window grows

### ISSUE THREAD

**Issue by zaza42** [bbidulock/icewm#512]
I just experienced a small bug with [gtkfilechooser](https://developer.gnome.org/gtk3/stable/GtkFileChooser.html) dialog, especially with chromium. When I'm browsing through images with thumbnails, the window size is constantly growing and growing on every movement. Here is a video: https://0x0.st/ini8.mp4
This is the behavior of earlier versions (til 1.9.0): https://0x0.st/iniN.mp4

**Comment by xtradev** [comment]
I just downgraded back to 1.9.0 because of similar experience with Calizo - a program I run via wine.
Also I dislike new proportions of confirmation dialogs in 1.9.1.

**Comment by xtradev** [comment]
And even in 1.9.0 try for example to run NEdit (a text editor). Its window opens as normal and then immediately jumps to something like 3x size of the screen. It does not do that in 1.8.3.

**Comment by ghost** [comment]
This is an issue for me as well, while using terminator, when i press tab it grows the window... along with my file extracting program, xarchiver, in that program pressing anything resizes it spastically. I'm heavily debating downgrading, although i don't know how easy that is with arch linux.

**Comment by teggun1** [comment]
Same issue here, i can see it with my graphical file manager pcmanfm, opening new tabs and closing/reopening the program causes it to take up more space each time, and some wine applications just keep increasing in size even when i just drag them around.

**Comment by gijsbers** [comment]
Commit e3dc92c fixes the slowly growing windows.
You can test this from git repository https://github.com/bbidulock/icewm/.
The website https://ice-wm.org/ explains how to compile.
If you would like me to test your case, then please provide descriptions of clear reproducable scenarios with steps.
Screenshots would also be nice.

**Comment by gijsbers** [comment]
Tests which now succeed for me: gtkfilechooser on tineye, xarchiver, NEdit, pcmanfm.

**Comment by zaza42** [comment]
ok, it seems to be fixed.

**Comment by xtradev** [comment]
In 1.9.1-2 NEdit and file choosers work fine, but wine windows keep increasing in size whenever I drag them around or just by themselves if something is happening inside them (like progress bar filling or MDI windows opening).

Also the confirmation dialogs in 1.9.1 to my eye look disproportionate. It was better before this version.

**Comment by gijsbers** [comment]
It is difficult to deduce from words what might be the problem. 
Could provide clear screenshots for what it used to be and what is now?
You should try the recently released icewm version 1.9.2.

**Comment by xtradev** [comment]
I am sorry. It seems Artix repositories did not catch up and I  installed some (intermediate?) version.
Everything in IceWM 1.9.2-1 works fine now. The confirmation dialogs look normal. Thank you.

```
$ yay -Si icewm
Repository      : world
Name            : icewm
Version         : 1.9.1-2
Description     : A Window Manager designed for speed, usability, and consistency
Architecture    : x86_64
URL             : https://ice-wm.org/
Licenses        : LGPL
Groups          : None
Provides        : None
Depends On      : libxrandr  libxinerama  libsndfile  libao  fribidi  libxft  libxcomposite  libxdamage  gdk-pixbuf-xlib  libxpm
Optional Deps   : perl: for icewm-menu-xrandr
Conflicts With  : None
Replaces        : None
Download Size   : 1660.70 KiB
Installed Size  : 5331.10 KiB
Packager        : Artix Build Bot <jenkins@artixlinux.org>
Build Date      : 2020-11-10T02:41:16 +07
Validated By    : MD5 Sum  SHA-256 Sum  Signature

Repository      : extra
Name            : icewm
Version         : 1.9.2-1
Description     : A Window Manager designed for speed, usability, and consistency
Architecture    : x86_64
URL             : https://ice-wm.org/
Licenses        : LGPL
Groups          : None
Provides        : None
Depends On      : libxrandr  libxinerama  libsndfile  libao  fribidi  libxft  libxcomposite  libxdamage  gdk-pixbuf-xlib  libxpm
Optional Deps   : perl: for icewm-menu-xrandr
Conflicts With  : None
Replaces        : None
Download Size   : 1660.29 KiB
Installed Size  : 5315.10 KiB
Packager        : Antonio Rojas <arojas@archlinux.org>
Build Date      : 2020-11-12T05:13:52 +07
Validated By    : MD5 Sum  SHA-256 Sum  Signature

```
